LovelyFox-code/star-wars
# implement index and detail page.

# What does this PR do?
- create design and layout of a single-page application with react create app;
- clicking on a film shows a new view of other details about the movie;
# Why?
- this is the first version of the app, that implements the requirements of a movie list page and detail page
- in the next versions I am going to improve the app
# How?
- create a design;
- create react app;
- create reusable components;
- module CSS for styles;
- fetch API with promises;
- navigate with react-router;
# How to test it?
- there is no automatic testing right now;
- click the button and see the result;
# Anything else?
- Currently, the app loads a little bit slow, because I fetch all the related data, I will try to change this in the next version
### gif preview:
#### mobile
![v1_mobile](https://user-images.githubusercontent.com/51757710/201078892-574bf570-a849-464e-9a77-16c22a750a36.gif)
#### desktop
![v1_demo](https://user-images.githubusercontent.com/51757710/201080749-69563610-7cd4-478f-8884-d388cfbd3157.gif)
